src/options/mkFlake.nix

```diff
@@ -252,6 +252,16 @@ let
           If enabled, IOGX will trace debugging info to standard output.
         '';
       };
+
+      sourceRepoUrl = l.mkOption {
```

Comment:
I think this should be part of `mkContainerFromCabalExe` and not `mkFlake`, since it's only used there, no?

Reply:
That is `sourceRepoUrl` should be part of `userConfig`

Reply:
Yeah I think that is a good idea. I was thinking that the user won't want to repeat the url for every container, but I think that issue should be solved latter. At least for now keeping it local `mkContainerFromCabalExe` leaves more options later on.